doc/src/Transactions.dox

```diff
@@ -82,7 +82,7 @@
  *    """
  *    with coordinator.scopedActionGroup(context):
  *        # Create the shot so we can publish
- *        shot_ref = manager.register([shot.name], [shot.metadata],
+ *        shot_ref = manager.register([shot.name], [shot.attributes],
  *                                    [shot.spec], [entity_ref], context)[0]
```

Comment:
[consider]/[plan] Not caused by this PR. This doesn't match the signature of register, attributes should be a keyword argument.

Reply:
#129